### ISSUE TITLE
Apps: thumbnail page navigation

### ISSUE THREAD

**Issue by asizemore** [VEuPathDB/web-eda#1121]
Currently the thumbnail page just lists all apps atop each other. If the user has lots of apps, this page can get quite lengthy. Let's help the user out by adding some navigation.

For example:
<img width="1139" alt="Screen Shot 2022-05-03 at 5 00 22 PM" src="https://user-images.githubusercontent.com/11710234/166566023-d9e477d1-c31e-44ec-8c42-9b1d68a87fc9.png">

In the above we have created four apps. Just above the navigation is the + New Module button. A new app would get added to the top of this nav list. Additionally, the "Some plots" app box is highlighted because that's where we are on the right - the Some plots app is at the top of the screen. If we scroll down to My happy app...

<img width="896" alt="Screen Shot 2022-05-03 at 5 01 15 PM" src="https://user-images.githubusercontent.com/11710234/166566279-e4db13ec-7aac-4876-a878-3903add82b86.png">

Alternatively, i could click any box in the navigation to jump right to the app. 

In general, the idea is basically the same as the left navigation on the genomics [record pages](https://vectorbase.org/vectorbase/app/record/dataset/DS_e2eead5bd4#Publications). In that navigation, it has a little blue dot next to the heading when that section of the page is at the top of the view, and each heading in the navigation is clickable and will take you to that part of the page. 

Currently let's just put the user-given app name in the button. We users will learn to name our apps :)

**Comment by asizemore** [comment]
Marking as "must have" for june 9 demo mostly because it helps the + New App button become really obvious. Certainly there are other ways to get around this if push comes to shove!

**Comment by asizemore** [comment]
Now that we have a different flow, I think this navigation is less crucial for the first roll-out. Marking as nice to have.

Additionally, the navigation panel should only have one button-thing for each app type. For example, there will only be one Alpha diversity button-thing in the navigation, even if the user has 60 alpha diversity configurations in their visualizations. See below.

<img width="1042" alt="Screen Shot 2022-05-13 at 11 28 46 AM" src="https://user-images.githubusercontent.com/11710234/168316771-3ac00864-e9e1-4b84-8538-e13f9401b75f.png">

Also adding to the styling team discussion because the way the navigation in the screenshot looks is kinda just like a list of buttons eek.

**Comment by asizemore** [comment]
Depends on #1095

**Comment by asizemore** [comment]
I think we can close this issue. After the latest app redesign the left navigation won't make so much sense. Let's see how users really do with having the current few apps and then re-evaluate if we truly need this type of navigation. I'd guess we'll learn that basic filtering/sorting on the apps would be more beneficial.
